mulog-prometheus/src/com/brunobonacci/mulog/publishers/prometheus.clj

```diff
@@ -41,31 +50,57 @@
           (publish-records! config (transform (map second items)))
           (rb/dequeue buffer last-offset)))))
 
-
   java.io.Closeable
-  (close [_]
-    ;; TODO:
-    ))
-
-
+  (close [_]))
 
-(def ^:const DEFAULT-CONFIG
+(def ^:private DEFAULT-CONFIG
   {:max-items     1000
    :publish-delay 100
-   ;; publish-method :push-gateway | :scrape
-   ;; push-gateway-endpoint ""
-   ;; :register    (...)
-   ;; ....
+   :registry      (reg/create-default)
+   :push-gateway  {:endpoint nil
+                   :job      nil
+                   :gateway  nil}
 
    ;; function to transform records
-   :transform     identity
-   })
-
-
+   :transform         identity
+   ;; function to transform metrics
+   :transform-metrics identity})
```

Comment:
please document what is the expected arguments and return format of this function and when should/could be used.